index.js

```diff
@@ -16,7 +16,17 @@ module.exports = class NitroBypass extends Plugin {
 		const currentUser = await getModule(['getCurrentUser']);
 
 		// spoof client side premium
-		currentUser.getCurrentUser().premiumType = 2;
+		this.log("Checking for currentUser...");
+		var x = setInterval(() => {
```

Comment:
The infinite loop should break after several tries in case it can not find the current user to prevent unnecessary spam. If possible, an event should be fired to change the premium type field instead, just after the current user has been set.